config/cloudformation.yaml

```diff
@@ -176,7 +176,7 @@ Resources:
         - Key: App
           PropagateAtLaunch: true
           Value: !Ref App
-      VPCZoneIdentifier: !Split [ ',', !ImportValue MobileAppsApiVPC-Subnets]
+      VPCZoneIdentifier: !Split [ ',', !ImportValue MobileAppsApiVPC-PublicSubnets]
```

Comment:
Why are these changes included? Has there been a rebase/merge against #1351 that's got muddled somewhere?

Reply:
needed to merge this in yesterday so I can deploy to CODE, I won't include those before merging this PR 👍